package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Healthy office cultures often acknowledge good work from their coworkers and peers. Award your peers with acknowledgements that act like coins/points in Slack when they do awesome things - and never let the acknowledgement of their good work get lost in the shuffle again.",
   "main": "index.js",
   "scripts": {
-    "server": "nodemon index.js",
```

Comment:
Have the app entry file in the root directory

Reply:
with the pattern discoursed below, we can't have that, sorry